src/server/supplements/recorder/javascript.ts

```diff
@@ -22,9 +22,16 @@ import { MouseClickOptions, toModifiers } from './utils';
 import deviceDescriptors from '../../deviceDescriptors';
 
 export class JavaScriptLanguageGenerator implements LanguageGenerator {
-  id = 'javascript';
-  fileName = '<javascript>';
+  id: string;
+  fileName: string;
   highlighter = 'javascript';
+  private _isTest: boolean;
+
+  constructor(isTest: boolean) {
+    this.id = isTest ? 'test' : 'javascript';
```

Comment:
```suggestion
    this.id = isTest ? 'playwright-test' : 'javascript';
```

to not clash with nunit in future